api/src/pcapi/routes/backoffice/templates/chronicles/list.html

```diff
@@ -0,0 +1,80 @@
+{% extends "layouts/connected.html" %}
+{% from "components/forms.html" import build_filters_form with context %}
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% from "components/collapsable_text.html" import collapsable_text with context %}
+{% block page %}
+  <div class="pt-3 px-5">
+    <h2 class="fw-light">Chroniques</h2>
+    <div class="filters">{{ build_filters_form(form, dst) }}</div>
+    <div>
+      {% if rows %}
+        <div class="d-flex justify-content-between">
+          <p class="lead num-results d-inline">{{ rows.total }} résultat{{ rows.total | pluralize }}</p>
+          <div>{% include "components/search/pagination.html" %}</div>
+        </div>
+        <table class="table mb-4">
+          <thead>
+            <tr>
+              <th scope="col"></th>
+              <th scope="col">ID</th>
+              <th scope="col">Titres des œuvres</th>
+              <th scope="col">Contenu</th>
+              <th scope="col">Date de création</th>
+              <th scope="col">Publiée</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for chronicle in rows.items %}
+              <tr>
+                <td>
+                  {% if has_permission("READ_CHRONICLE") %}
```

Comment:
Probablement plutôt : 
```suggestion
                  {% if has_permission("MANAGE_CHRONICLE") %}
```